modular_skyrat/modules/automapper/readme.md

```diff
@@ -0,0 +1,12 @@
+## Credits 
+Code: [@Gandalf2k15]
+
+## About
+This automapper is designed to make mapping far easier for modular maps, it should mean less manual map resets(ideally, never).
+It supports two types of automapping, simple area automapper and template automapper.
+
+## Simple Area Automapper
+What this will do is check an area for a suitable single tile to spawn an object in. It uses "intelligent" heatmapping to choose a suitable place.
```

Comment:
It would be nice for you to define what "intelligent" means here, just so it's easier to understand without looking too deep into the code in the future.